### PR TITLE
add unit test to kafkaconfigmap

### PR DESCRIPTION
**Pull request by mgold1234**
Signed-off-by: mgold1234 <mgold@redhat.com>
related to ticket https://issues.redhat.com/browse/THEEDGE-2952 

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by de1987 — DISMISSED** [review]

File: pkg/common/kafka/kafkaconfigmap_test.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+	// Reset config.kafkaconfig back to its original value
+	defer func(conf *v1.KafkaConfig) {
+		config.Get().KafkaConfig = conf
+	}(conf)
+
+	ctx := context.Background()
+	cfg.KafkaConfig = &v1.KafkaConfig{}
+	cfg.KafkaConfig.Brokers = []clowder.BrokerConfig{brokerConfig}
+	cfg.KafkaConfig.Brokers = append(cfg.KafkaConfig.Brokers, brokerConfig)
```
Comment (ldjebran) [inline pkg/common/kafka/kafkaconfigmap_test.go:57]: brokerConfig is already in the slice why we need add it again ?

**Review by ldjebran — COMMENTED** [review]
looks good , left one question

**Review by ldjebran — APPROVED** [review]
lgtm

**Review by de1987 — APPROVED** [review]

**Comment by omaciel** [comment]
@Mergifyio rebase

**Pull request merged**